src/Routing/Filter/ThrottleFilter.php

```diff
@@ -22,28 +49,18 @@ public function __construct($config = [])
         $config += [
             'priority' => 1,
             'message' => 'Rate limit exceeded',
-            'interval' => '1 minute',
+            'interval' => '+1 minute',
             'rate' => 10,
             'identifier' => function (Request $request) {
                 return $request->clientIp();
             },
+            'headers' => true,
+            'rateHeader' => 'X-Throttle-Limit',
+            'remainingHeader' => 'X-Throttle-Remaining',
+            'resetHeader' => 'X-Throttle-Reset'
```

Comment:
Should headers be instead accept either false or an array (empty array same as false) and have the 'header' related elements defined in the headers array?